# origin: Debian (herbert)
# cset: n/a
# inclusion: not submitted
# description: ECN is always available as a sysctl, so this check is wrong.
# revision date: 2004-09-03

diff -urN kernel-source-2.4.26/net/ipv4/tcp_diag.c kernel-source-2.4.26-1/net/ipv4/tcp_diag.c
--- kernel-source-2.4.26/net/ipv4/tcp_diag.c	2003-08-25 21:44:44.000000000 +1000
+++ kernel-source-2.4.26-1/net/ipv4/tcp_diag.c	2003-09-03 20:27:21.000000000 +1000
@@ -155,10 +155,8 @@
 			info->tcpi_snd_wscale = 0;
 			info->tcpi_rcv_wscale = 0;
 		}
-#ifdef CONFIG_INET_ECN
 		if (tp->ecn_flags&TCP_ECN_OK)
 			info->tcpi_options |= TCPI_OPT_ECN;
-#endif
 
 		info->tcpi_rto = (1000000*tp->rto)/HZ;
 		info->tcpi_ato = (1000000*tp->ack.ato)/HZ;
